app/gui/view/welcome-screen/src/lib.rs

```diff
@@ -103,13 +109,21 @@ impl Model {
         Self { application, logger, dom, display_object, side_menu, template_cards }
     }
 
+    /// Set displayed projects list.
+    pub fn set_projects_list(&self, projects: &[String]) {
+        self.side_menu.set_projects_list(projects);
+    }
+
     fn create_dom(
         logger: &Logger,
         side_menu: &SideMenu,
         template_cards: &TemplateCards,
     ) -> DomSymbol {
         let root = web::create_div();
         root.set_class_name(css_class::TEMPLATES_VIEW_ROOT);
+        // We explicitly enable pointer events for Welcome Screen elements. Pointer events are
+        // disabled for all DOM layers by default.
```

Comment:
Add a reference where to find explanation _why_ DOM layers have disabled pointer events.

Reply:
Done.